scripts-local/tests/integration_tests.bats

```diff
@@ -44,7 +58,7 @@ setup() {
 ######                              SETUP TESTS                                         ######
 ##############################################################################################
 
-@test "address names are correct" {
+@test "[INTEGRATION-BASIC] address names are correct" {
```

Comment:
nit: can we add the ICA addresses here as well and then reference them in "host zone successfully registered"?

Reply:
Added to vars.sh but assert_line won't let me pipe them in with `$variable` for some reason (syntax issue)? Will put this on the backlog as a nice to have.

Reply:
It might be because you're using single quotes in the register host zone test? A single quote in bash doesn't expand the variable. 
Ex: 
```
var="hello"

echo "$var"
>>> hello

echo '$var'
>>> $var
```

Also my comment was a little unclear but I was saying we should add those addresses to this "address names are correct" test as well! In the same way that we validate stride/gaia/icq addresses. 
I thought we ran into a case when testing at one point where our tests were broke cause these addresses got changed, so I was thinking it'd be nice to double check em here so it doesn't happen again!

Reply:
Great point. Added!